### ISSUE TITLE
Doubt some changes on 79fbd3b (cast times)

### ISSUE THREAD

**Issue by cydh** [rathena/rathena#1061]
There is a point on 79fbd3bc699beb6d879c92d4342aa073a036c9f6

> Updated all non-3rd class skills cast times. (https://github.com/HerculesWS/Hercules/commit/e2fdc3b858dda8abb7518735fbb254a2d63ea760)

I don't check and test for all skills, just, I noticed that the changes for **Hunter's Traps** are off.
Based on I experienced on idRO (and just tested on Aegis server), the trap skills **DO NOT HAVE** cast time and fixed cast time.

So, @aleos89 @Playtester @Lemongrass3110 maybe you can review Aegis cast time 'again'?

**Comment by Playtester** [comment]
Our sources are outdated.

14.1 rebalance patch increased cast time and aftercast delays of most skills. Many official servers reverted these changes, though. But since we are kRO based...

Read for example here:
https://forums.warpportal.com/index.php?/topic/62368-skill-cast-requests/

> [Hunter]
> Claymore Trap, Blast Mine, Land Mine
> Pre-patch times - Instant Cast
> Post-patch times - Some somewhat long cast time

**Comment by Playtester** [comment]
I'm not sure how reliable Herc's data is. Like did they really convert an official file or just got some info from testing?

I noticed I have reverted some of these changes too.

For example that update changes Raging Palm Strike from having an aftercast delay to having a cast time. I reverted that.

And the update changed Sightrasher from 400ms variable cast to 80ms fixed cast which I then again updated to 320ms variable cast and 80ms fixed cast.

Hmmm...
